frontend/src/services/backend_service_interface/backend_service_interface.ts

```diff
@@ -5,6 +5,7 @@ export interface BackendProfileServiceInterface {
 }
 
 export interface BackendYourClubsServiceInterface {
+  createClub(clubJson: string): Promise<boolean>;
   listClubs(numClubs: number): Promise<string>;
   leaveClub(clubId: string): Promise<boolean>;
 }
```

Comment:
In this PR, and throughout the codebase in general, interfaces are defined with lines ending in `;` or `,`, we should probably just stick to one (seems like most follow `,`) and from here on out change any instances that we see that break that rule.